build.sbt

```diff
@@ -136,11 +136,7 @@ lazy val `sbt-plugin` = myCrossProject("sbt-plugin")
 
 lazy val `mill-plugin` = myCrossProject("mill-plugin")
   .settings(
-    crossScalaVersions := Seq(Scala213, Scala212),
```

Comment:
I think `crossScalaVersions` is required so that the mill plugin is published for Scala 2.12 and 2.13.

Reply:
Removed by accident. Fixed now.